src/__tests__/lib/containers/SynapseTable.test.tsx

```diff
@@ -31,11 +31,18 @@ import syn16787123Json from '../../../mocks/syn16787123.json'
 import { cloneDeep } from 'lodash-es'
 import HasAccess from '../../../lib/containers/HasAccess'
 import { NOT_SET_DISPLAY_VALUE } from '../../../lib/containers/table/SynapseTableConstants'
+import { SynapseContextProvider } from '../../../lib/utils/SynapseContext'
+import { MOCK_CONTEXT_VALUE } from '../../../mocks/MockSynapseContext'
 
 const createShallowComponent = (
   props: SynapseTableProps & QueryWrapperChildProps,
 ) => {
-  const wrapper = mount<SynapseTable>(<SynapseTable {...props} />)
+  const wrapper = mount<SynapseTable>(<SynapseTable {...props} />, {
+    wrappingComponent: SynapseContextProvider,
+    wrappingComponentProps: {
+      synapseContext: MOCK_CONTEXT_VALUE,
+    },
+  })
```

Comment:
TODO: Should use this style for all mounted components. I think this is preferable because `wrapper.instance()` just works. It probably works better with `wrapper.setContext()` as well